ccgains/trades.py

```diff
@@ -257,9 +267,43 @@ def _parse_trade(str_list, param_locs, default_timezone):
         else:
             pdict[key] = val
 
+    # Get missing fees if requested
+    if fill_missing_fees and pdict['kind'] == 'Withdrawal':
+        tx = pdict['mark']
+        fee_curr = pdict['fee_currency']
+        pdict['fee_amount'] = _get_tx_fees(tx, fee_curr)
+
     return Trade(default_timezone=default_timezone, **pdict)
 
 
+def _get_tx_fees(txid, currency):
+    """Get missing transaction fees using blockcypher API.
+
+    :param tx: str:
+        The transaction-ID.
+    :param currency: str:
+        The currency of the fees.
+
+    :return: obtained fees as Decimal object
+
+    """
+    from blockcypher import get_transaction_details, from_satoshis
+    try:
+        tx = get_transaction_details(txid, coin_symbol=currency.lower())
+        if currency == 'BTC':
+            fee = from_satoshis(tx['fees'], currency.lower())
+            feeval = Decimal(fee)
+        elif currency == 'LTC':
+            # I am not sure about this factor, but it
+            # seems to work
+            fee = tx['fees'] * 0.00000001
```

Comment:
This factor is correct (see my comment w/ screenshot on the PR discussion), but to make it more explicit, perhaps change to:

```suggestion
            fee = tx['fees'] / 10**8  # Smallest unit of measure for LTC
```